test/index.js

```diff
@@ -760,6 +761,172 @@ describe('normalizr', function () {
       }
     });
   });
+  
+  it('fails normalizing with a relationship', function () {
+    var user = {
+      id: 1,
+      relations: []
+    };
+      
+    (function () {
+      normalize(user, relationOf('userRelations'))
+    }).should.throw('Normalize does not accept a relation as a schema');
+  });
+
+  it('can normalize nested object with relationship', function() {
+    var job = new Schema("jobs");
+    var user = new Schema('users');
+    user.define({
+      relations: relationOf("userRelations", {
```

Comment:
I don't understand why this an't just be `arrayOf`. That way you keep your relationship data on the object in question.